io.github.prateekmedia.appimagepool.yml

```
@@ -0,0 +1,58 @@
+app-id: io.github.prateekmedia.appimagepool
+runtime: org.gnome.Platform
+runtime-version: "40"
+sdk: org.gnome.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.rust-stable
+rename-icon: appimagepool
+rename-appdata-file: appimagepool.appdata.xml
+rename-desktop-file: appimagepool.desktop
+command: appimagepool
+separate-locales: false
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --socket=pulseaudio
+  - --share=network
+  - --device=dri
+  - --persist=Applications
+  - --talk-name=org.freedesktop.Notifications
+build-option:
+  prepend-path: /app/lib/sdk/flutter/bin
+  append-path: /usr/lib/sdk/rust-stable/bin
```

```suggestion
  append-path: /usr/lib/sdk/rust-stable/bin:/app/lib/sdk/flutter/bin
```

this ought to be enough